dcgmlib/src/DcgmClientHandler.cpp

```diff
@@ -229,8 +229,8 @@ dcgmReturn_t DcgmClientHandler::GetConnHandleForHostEngine(const char *identifie
     unsigned int portNumber = 0;
     char *p                 = NULL;
 
-    if (!addressIsUnixSocket)
```

Comment:
Can we add a comment about intent of new check and maybe existing check if you are aware of it?